desktop_version/src/Game.cpp

```diff
@@ -336,7 +336,8 @@ void Game::init(void)
     stat_trinkets = 0;
 
     state = 1;
-    statedelay = 0;
+    setstatedelay = 0;
```

Comment:
No, this is incorrect. `setstatedelay` is a function, not a variable.
```suggestion
    statedelay = 0;
```

Reply:
agh, sorry, was distracted while committing

Reply:
lmao